app/models/notification.rb

```diff
@@ -75,7 +75,9 @@ def self.ensure_consistency!
     DB.exec(<<~SQL)
       DELETE
         FROM notifications n
-       WHERE high_priority AND notification_type <> #{types[:chat_mention].to_i}
+       WHERE high_priority
+         AND notification_type <> #{types[:chat_mention].to_i}
+         AND notification_type <> #{types[:chat_message].to_i}
```

Comment:
`AND notification_type NOT IN (#{types[:chat_mention].to_i}, #{types[:chat_message].to_i})`